Example/Tests/Classes/EventDataManagerTest.m

```diff
@@ -90,11 +90,6 @@ - (void)test_migrateRequests
     [[LeanplumRequest sendNowQueue] cancelAllOperations];
     [[LeanplumRequest sendNowQueue] waitUntilAllOperationsAreFinished];
     
-    NSFileManager *fileManager = [NSFileManager defaultManager];
```

Comment:
are you sure we don't need this ? This makes sure we always start without present DB

Reply:
Yeah in iOS11 it seems like we can't remove sqlite db. It gives an I/O exception. Which we track and the event count goes up to 2.